api/envoy/service/auth/v2alpha/external_auth.proto

```diff
@@ -24,21 +25,37 @@ message CheckRequest {
   AttributeContext attributes = 1;
 }
 
+// Http attributes for denied responses.
+message DeniedHttpResponse {
+  // Http status code.
+  envoy.api.v2.core.HttpStatus status = 1;
+
+  // Http entity headers.
+  map<string, string> headers = 2;
+
+  // Http entity body.
+  string body = 3;
+}
+
+// Http attributes for ok responses.
+message OkHttpResponse {
+  // Http entity headers.
```

Comment:
These headers are in addition to existing headers or in addition to existing headers. A comment would help. Thanks.